package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "thrift-pool",
```

Comment:
the name `thrift-pool` is taken in npm: https://www.npmjs.com/package/thrift-pool